test/distributed/org/apache/cassandra/distributed/impl/InstanceConfig.java

```diff
@@ -114,6 +115,47 @@ private InstanceConfig(int num,
                 .set("counter_cache_size", "50MiB")
                 .set("key_cache_size", "50MiB")
                 .set("commitlog_disk_access_mode", "legacy");
+        if (CassandraRelevantProperties.DTEST_JVM_DTESTS_USE_LATEST.getBoolean())
+        {
+            // TODO: make this load latest_diff.yaml or cassandra_latest.yaml
```

Comment:
This TODO seems quite critical. Wouldn't we prefer a crude yaml read for the param values rather than hard-coded?

Reply:
Not if the non-latest path does not read any configuration file either.

Both need to be fixed to read a config file (possibly applying changes on top of it), but my quick attempt to do it failed and I don't want to delay the introduction and testing of the "latest" config (and 5.0 with it).

Reply:
Ok let's create a ticket for this so it doesn't fall through the cracks.